### PR TITLE
Week3

### PR DESCRIPTION
**Pull request by arnavss2**
Add some styling for UI
Add variables for constants

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/Contact.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    private long id;
+    private String name;
+    private String email;
+    private String message;
+
+    public Contact(){
+        
+    }
+
+    public Contact(long set_id, String set_name, String set_email, String set_message){
```
Comment (dchickabasapa) [inline portfolio/src/main/java/com/google/sps/servlets/Contact.java]: Have this code as:

public Contact(long id, String name, String email, String message) {
  this.id = id;;
  this.name = name;
  ....
}

File: portfolio/src/main/java/com/google/sps/servlets/DeleteContactServlet.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for deleting contacts. */
+@WebServlet("/delete-contact")
+public class DeleteContactServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```
Comment (dchickabasapa) [inline portfolio/src/main/java/com/google/sps/servlets/DeleteContactServlet.java]: Delete should be a doDelete rather than a doPost.

**Review by dchickabasapa — CHANGES_REQUESTED** [review]
Would like another iteration as there is a logical mistake. 

Also remove an empty file in the change.

**Comment by arnavss2** [comment]
I have the empty file tailwind.config.js because it is required for using IntelliSense in VSCode.

**Review by dchickabasapa — APPROVED** [review]

**Pull request merged**